spec/lib/centralized_metadata/macros/custom_spec.rb

```diff
@@ -288,4 +288,33 @@
       end
     end
   end
+
+  describe "add_filename" do
+    before(:each) do
+      indexer.configure do
+        to_field "cm_filename", add_filename
+      end
+    end
+
+    context "when filename override provided" do
+      it "will set cm_source_vocab to lcsh" do
```

Comment:
@dkinzer I'm confused about this line.  Is this supposed to be cm_filename rather than cm_source_vocab?  We already have tests for that field.